cabra/src/pages/components/TaskRelationsEdit.tsx

```diff
@@ -0,0 +1,89 @@
+import tw, { styled } from "twin.macro";
+
+import { DirectedRelationType } from "../../types/taskRelation";
+import { ITask } from "../../types/task";
+import RelationListEdit from "./RelationListEdit";
+import Select from "react-select";
+import { getTasks } from "../../api/tasks";
+import { useQuery } from "react-query";
+import { useState } from "react";
+
+const Label = styled.label(tw`text-gray-50 font-bold`);
+const CenteredLabel = styled.div(tw`text-gray-50 font-bold text-center`);
+const Container = styled.div(tw`text-gray-50 w-full space-y-4`);
+interface Props {
+  readonly taskId: string;
+}
+interface RelationshipOption {
+  readonly value: DirectedRelationType;
+  readonly label: string;
+}
+interface TaskOption {
+  value: ITask;
+  label: string;
+}
+
+const relationshipOptions: RelationshipOption[] = [
+  { value: DirectedRelationType.Subtask, label: "Subtask" },
+  { value: DirectedRelationType.DependsOn, label: "Depends on" },
+  { value: DirectedRelationType.IsDependentOn, label: "Is dependent on" },
+  { value: DirectedRelationType.Interchangable, label: "Interchangeable" },
+];
+
+const DEFAULT_RELATION_TYPE = DirectedRelationType.Subtask;
+
+export default function TaskRelationsEdit({ taskId }: Props) {
+  const [tasksRelation, setTaskRelation] = useState<DirectedRelationType>();
```

Comment:
```suggestion
  const [tasksRelation, setTaskRelation] = useState<DirectedRelationType>(DEFAULT_RELATION_TYPE);
```
or `null`
This value should never be undefined